source/includes/documentation/_vue_js_elements.md

```diff
@@ -2608,50 +2608,54 @@ For examples of usage see: [Menus tests](https://github.com/jdi-testing/jdi-ligh
 
 ### 5.29 Images
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/images/)
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/images/)
 
 - __Java__: _package com.epam.jdi.light.vuetify.elements.common.Image.java_
 
 ```java
+    //@FindBy(css = "#GradientsImage .v-image")
+    @UI("#GradientsImage")
+    public static Image gradientsImage;
+    
     @Test
```

Comment:
please, use, the final version where tests are with descriptions :)

Reply:
Fixed